lib/checks/language/valid-lang.tpl

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file isn't really necessary, you can rather easily read the JSON file, change options and then save it again.  If you decide to keep this file, then the valid-lang.json file should be removed from source control.